libcodechecker/analyze/analyzer.py

```diff
@@ -178,6 +178,35 @@ def perform_analysis(args, context, actions, metadata):
         statistics_data = manager.dict({'stats_out_dir':
                                         args.stats_output})
 
+    if 'stats_min_sample_count' in args:
+        try:
+            if int(args.stats_min_sample_count) > 1:
+                statistics_data['stats_min_sample_count'] =\
+                    args.stats_min_sample_count
+            else:
+                LOG.error("stats_min_sample_count"
+                          "must be greater than 1.")
+                return
+        except ValueError:
+            LOG.error("stats-min-sample-count"
+                      "must be an integer.")
+            return
+
+    if 'stats_relevance_threshold' in args:
```

Comment:
If I remember it correctly the default value will be always set so no checking is needed.